src/middlewares/index.ts

```diff
@@ -0,0 +1,2 @@
+export { createInsightsMiddleware } from './insights';
```

Comment:
To be consistent with the file naming, we can rename the file `createInsightsMiddleware` (and perhaps `createRouterMiddleware`?).

Reply:
https://github.com/algolia/instantsearch.js/pull/4446/commits/183b1ca2e8f7727ba397a1cb459975fd6ddfa9d9 good call!